src/transforms/interval.js

```diff
@@ -59,7 +49,13 @@ function maybeIntervalMidK(k, maybeInsetK, options) {
     ...options,
     [k]: {
       label: labelof(v),
-      transform: data => valueof(data, value).map(interval.mid)
+      transform: data => {
+        const V1 = valueof(data, value).map(v => interval.floor(v));
+        const V2 = V1.map(v => interval.offset(v));
+        return V1.map(isTemporal(V1)
+          ? (v1, i) => new Date((+v1 + +V2[i]) / 2)
+          : (v1, i) => (+v1 + +V2[i]) / 2);
```

Comment:
This probably needs to handle `v1` or `V2[i]` being null here. I’d guess in that case the transform should produce undefined for dates (to match coerceDate) and NaN for numbers (to match coerceNumber).

Reply:
I’ve added a commit that handles missing data here. But, it occurs to me that interval.floor and interval.offset won’t handle missing data, either, so… 🤷